tox.ini

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
But how do you test for both python 2 and 3? Do you use two distinct virtualenvs, one for each version?